### PR TITLE
[16.0][ADD] mail_send_confirmation

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
This module asks for confirmation when 'Send' button in the message composer of the chatter is pressed, to reduce the chances of accidentally sending an internal message to the external followers.

@qrtl

### CODE REVIEW

**Comment by rinaldifirdaus** [comment]
Functional test: Works as expected!

File: mail_send_confirmation/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+This module asks confirmation when sending mail to the followers.
```
Comment (yostashiro) [inline mail_send_confirmation/readme/DESCRIPTION.rst]:
```suggestion
This module asks for confirmation when 'Send' button in the message composer of the chatter is pressed, to reduce the chances of accidentally sending an internal message to the external followers.

Limitation
~~~~~~~~~~

As of now, this module does not change the behavior of the full composer (i.e. no confirmation will be requested), which shows the recipients and therefore the extra confirmation step may not be as necessary as in the simple composer.
```

Let's add more information.

**Comment by yostashiro** [comment]
@AungKoKoLin1997 Thanks! Can you please do the same for https://github.com/OCA/social/pull/1074.

**Review by rinaldifirdaus — APPROVED** [review]

**Review by CasVissers-360ERP — APPROVED** [review]
Functional review,  nice little addon!

**Comment by AungKoKoLin1997** [comment]
@OCA/social-maintainers Is it possible to take a look at this PR?

**Comment by CasVissers-360ERP** [comment]
@OCA/social-maintainers reminder

**Review by dreispt — APPROVED** [review]

**Comment by AungKoKoLin1997** [comment]
@dreispt I think merge was failed because of missing `__init__.py `. I added it. Can you please try again?

**Pull request merged**